sdks/java/container/boot_test.go

```diff
@@ -78,3 +78,15 @@ func TestBuildOptions(t *testing.T) {
 		t.Errorf("BuildOptions(%v).JavaProperties = %v, want %v", metaOptions, javaOptions.Properties, wantProperties)
 	}
 }
+
+func TestHeapSizeLimit(t *testing.T) {
+	if lim := HeapSizeLimit(0); lim != 1 << 30 {
+		t.Errorf("HeapSizeLimit(0). Actual (%d). want 1 GB", lim)
+	}
+	if lim := HeapSizeLimit(2 << 30); lim != (2 << 30) * 7 / 10 {
+		t.Errorf("HeapSizeLimit(0). Actual (%d). want 700 MB", lim)
```

Comment:
The parameter wasn't updated in the t.Error call. Here and the last one too.

Reply:
sands for catching it. fixed